assets/src/blocks/Spreadsheet/SpreadsheetBlock.js

```diff
@@ -0,0 +1,58 @@
+import {SpreadsheetEditor} from './SpreadsheetEditor';
+import {CSS_VARIABLES_ATTRIBUTE} from '../CssVariablesAttribute';
```

Comment:
I think that this file (`CssVariablesAttribute`) should be in the `Spreadsheet` folder, it won't be used by any other block

Reply:
It could even be defined directly here in this file 🤔 It's not needed anywhere else, and it's quite small